### PR TITLE
Gayles Job Process

### PR DESCRIPTION
**Pull request by john0005**

### CODE REVIEW

File: src/main/java/com/esri/samples/localserver/local_server_geoprocessing/LocalServerGeoprocessingController.java [hunk 1]
```diff
@@ -135,12 +145,10 @@ protected void handleGenerateContours() {
         mapView.getMap().getOperationalLayers().add(mapImageLayer);
         btnGenerate.setDisable(true);
       } else {
-        Platform.runLater(() -> {
```
Comment (ghost) [inline src/main/java/com/esri/samples/localserver/local_server_geoprocessing/LocalServerGeoprocessingController.java:138]: Has a failed job been tested to see if this is required?

File: src/main/java/com/esri/samples/localserver/local_server_geoprocessing/LocalServerGeoprocessingController.java [hunk 2]
```diff
@@ -125,6 +131,10 @@ protected void handleGenerateContours() {
 
     // adds contour lines to map
     GeoprocessingJob gpJob = gpTask.createJob(gpParameters);
+    gpJob.addProgressChangedListener(() -> {
```
Comment (ghost) [inline src/main/java/com/esri/samples/localserver/local_server_geoprocessing/LocalServerGeoprocessingController.java]: Can remove the braces.

File: src/main/java/com/esri/samples/localserver/local_server_geoprocessing/LocalServerGeoprocessingController.java (hunk 1 above)
Comment (john0005) [reply src/main/java/com/esri/samples/localserver/local_server_geoprocessing/LocalServerGeoprocessingController.java:138]: I tested it and worked without.

Comment (gayl0000) [reply src/main/java/com/esri/samples/localserver/local_server_geoprocessing/LocalServerGeoprocessingController.java:138]: yes, it usually does work without, that doesnt mean it is on the correct thread though. An API fix has gone in recently, so I should be able to remove this.

**Pull request merged**